apps/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -1,35 +1,34 @@
-import { Component, Input } from '@angular/core';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { StudentStore } from '../../data-access/student.store';
 import { TeacherStore } from '../../data-access/teacher.store';
 import { CardType } from '../../model/card.model';
 
 @Component({
   selector: 'app-list-item',
   template: `
+    <!-- <ng-template #appListItemTemplate let-item="item"> -->
     <div class="border border-grey-300 py-1 px-2 flex justify-between">
       {{ name }}
       <button (click)="delete(id)">
         <img class="h-5" src="assets/svg/trash.svg" />
       </button>
     </div>
```

Comment:
ng-content will create more flexibility to your component

Reply:
yeah seen :) thanks 👍